joomla/joomla-cms
# [Sql azure] Normalize update sql files

### Summary of Changes

Normalizes all sql azure update files.
- always use `SET IDENTITY_INSERT [#__xxxx] ON/OFF;` when inserting (like joomla.sql)
- `INSERT` to `INSERT INTO` (like joomla.sql)
- Quote all table names and db field names
- Always use `'` to quote string values.
- Code style
### Testing Instructions

Mainly code review. (easier to check code diff without white spaces changes) https://github.com/joomla/joomla-cms/pull/11722/files?w=1
If anyone could test and upgrade from 2.5.28 to 3.6.x would be great. Don't have sql azure to test,
### Documentation Changes Required

None.
